src/components/Banners/Banner/Banner.tsx

```diff
@@ -0,0 +1,70 @@
+//- React Imports
```

Comment:
A generic banner component. The idea would be to reuse this component for all instances of a banner. Data (title, image url, label etc) would be passed down to this component by requested data from API or json file etc.